grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc";
+
+service ConfigService {
+  rpc GetConfig (GetConfigRequest) returns (GlobalConfig);
+  rpc UpdateConfig (UpdateConfigRequest) returns (UpdateConfigResponse);
+  rpc CheckConfigGeneration (CheckConfigGenerationRequest)
+    returns (CheckConfigGenerationResponse);
+}
+
+message GetConfigRequest {
+}
+
+message UpdateConfigRequest {
+  GlobalConfig config = 1;
+}
+
+message UpdateConfigResponse {
+  string message = 1;
+  Error error = 2;
+}
+
+enum Error {
+  ERROR_NONE_UNSPECIFIED = 0;
+  ERROR_VALIDATION_FAILED = 1;
+  ERROR_APPLY_FAILED = 2;
+  ERROR_UNKNOWN_ERROR = 3;
+}
+
+message CheckConfigGenerationRequest {
+}
+
+message CheckConfigGenerationResponse {
+  uint64 generation = 1;
+}
+
+message Subnet {
+  string name = 1;
+  string prefix = 2;
+  uint32 vni = 3;
+}
+
+message VPC {
```

Comment:
(optional) could benefit from some ordering like: id, name, vrf, vni, subnets or smth like this